src/strikethrough.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license Copyright (c) 2017, CKSource - Rémy Hubscher. All rights reserved.
```

Comment:
We are not modifying standard header for each developer. "CKSource - Frederico Knabben" it's our company full name. We really appreciate your contribution, it will be noted in GitHub history and our change log, but we cannot change copyright information - this is also why we needed CLA to be signed.

Reply:
@Natim - are you ok with me modifying headers back to original state and merging this PR?

Reply:
Sorry I forgot about that, it should be fixed now.